_CoqProject

```diff
@@ -53,7 +50,6 @@ theories/Data/ListNth.v
 theories/Data/List.v
 theories/Data/Member.v
 theories/Data/Nat.v
-theories/Data/N.v
```

Comment:
It might make sense to keep this file, despite the fact that it is empty...I'm not certain.